samDeploy/README.md

```diff
@@ -0,0 +1,32 @@
+# Working with SAM
+
+## Setup
+To set up sam and be able to run it in the correct environment, run the following commands:
+
+Run these
+```shell
+brew install gradle@7
+echo '# Add gradle 7 to PATH' >> ~/.zshrc
+echo 'export PATH="/opt/homebrew/opt/gradle@7/bin:$PATH"' >> ~/.zshrc
+echo '' >> ~/.zshrc
+
+brew install jenv
```

Comment:
Can gradle just do this for us? https://docs.gradle.org/current/userguide/toolchains.html